CompanyEmployees.Core.Domain/Exceptions/AttendanceAlreadyExistsException.cs

```diff
@@ -0,0 +1,9 @@
+namespace CompanyEmployees.Core.Domain.Exceptions;
+
+public sealed class AttendanceAlreadyExistsException : BadRequestException
+{
```

Comment:
Consider adding an XML documentation comment for the exception class. This helps maintainers understand its purpose and improves generated documentation.

**Suggested change**
```suggestion
/// <summary>
/// Exception thrown when an employee attempts to check in more than once on the same day.
/// </summary>
public sealed class AttendanceAlreadyExistsException : BadRequestException
```